src/detections/fetch/repository/query.js

```diff
@@ -0,0 +1,21 @@
+import { gql } from '@apollo/client'
+
+export const GET_DETECTIONS = gql`
```

Comment:
Let's format this GraphQl query to make it more readable .

Reply:
Done